src/css/feature-cards.css

```diff
@@ -0,0 +1,48 @@
+.card-deck {
+	margin: 0 auto;
+	max-width: 1400px;
+}
+
+.card {
```

Comment:
I'd go with `.feature-card` vs `.card` to leave us open for a more generic `.card` one day. :)